Source/WebKit/WebProcess/WebPage/WebPage.cpp

```diff
@@ -966,6 +966,9 @@ WebPage::WebPage(PageIdentifier pageID, WebPageCreationParameters&& parameters)
         sandbox_enable_state_flag("EnableExperimentalSandboxWithProbability", *auditToken);
     }
 #endif // USE(APPLE_INTERNAL_SDK)
+
+    if (WebProcess::singleton().isLockdownModeEnabled())
```

Comment:
We use a different process ID for Lockdown Mode -- could we use bundle identifier instead of an entitlement? Or could we just test for `com.apple.security.cs.allow-jit`, like you do in the sandbox?

Reply:
> We use a different process ID for Lockdown Mode -- could we use bundle identifier instead of an entitlement? Or could we just test for `com.apple.security.cs.allow-jit`, like you do in the sandbox?

I think calling isLockdownModeEnabled() may be faster, since it only returns a boolean flag. Or perhaps I misunderstood the question? 

Thanks for reviewing!